osu.Game/Screens/Edit/Screens/Setup/Components/SwitchButton.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Input.States;
+using osu.Game.Graphics;
+
+namespace osu.Game.Screens.Edit.Screens.Setup.Components
+{
+    public class SwitchButton : CircularContainer, IHasCurrentValue<bool>, IHasAccentColour
```

Comment:
Shouldn't be a container itself.

Reply:
What would be best for it to be? According to the design it has to have that circular appearance, which I think only the `CircularContainer` would match.

Reply:
I just mean that this should be a `CompositeDrawable`, and should contain the `CircularContainer`.